plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClient.java

```diff
@@ -652,6 +653,14 @@ public JdbcOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
         }
     }
 
+    @Override
+    protected void verifyTableNameLength(DatabaseMetaData databaseMetadata, String tableName)
```

Comment:
What would happen if we did not validate anything here?
would Phoenix server reject the CREATE TABLE?

Reply:
Phoenix behavior is little tricky. I haven't investigated the border number. 
* `65536` chars: reject creation
* `32767` chars: cause timeout

Reply:
feel free to leave just a TODO for Phoenix and follow-up separately.